xml/System.Net/Cookie.xml

```diff
@@ -968,7 +968,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Net.Cookie.Value%2A> of a <xref:System.Net.Cookie> must not be `null`. The following characters are reserved and cannot be used for this property: semicolon, comma.  
+ The <xref:System.Net.Cookie.Value%2A> of a <xref:System.Net.Cookie> must not be `null`. If a `null` value is passed into the setter it will be replaced with an empty String. The following characters are reserved and cannot be used for this property: semicolon, comma. The setter won't throw an exception if invalid characters are passed into it but when you try to add this <xref:System.Net.Cookie> to a <xref:System.Net.CookieContainer> instance with the <xref:System.Net.CookieContainer.Add%2A> method the operation will fail and throw an exception.
```

Comment:
First, and most importantly, do you have any code that illustrates the exception? I can't reproduce the exception with simple code like the following, if run on either the .NET Framework or .NET Core:

using System;
using System.Net;

public class Example
{
   public static void Main()
   {
      var cookie = new Cookie();
//      cookie.Value = $"{DateTime.UtcNow}; Key=name,Value=SomeName;";
      cookie.Value = ";";
      var coll = new CookieCollection();
      coll.Add(cookie);
      Console.WriteLine($"{coll.Count} cookies in the collection:");
      foreach (Cookie item in coll)
         Console.WriteLine(item.Value);
   }
}
```
If you can supply some code that illustrates the exception, we can include it with the documentation.

Generally, instead of referring to setters, it's best to refer to property assignments. So, "If a `null` value is assigned to this property, it is replaced with an empty string." (Note the comma after property, as well as lowercase string.)

The sentence beginning "The following characters..." should begin a new paragraph; it's unrelated to null values.

Again, to eliminate the setter from the sentence "The setter won't throw an exception if invalid characters are passed into it but..." --> "Assigning invalid characters to this property does not throw an exception, but ..."

Finally, a comma before "the operation will fail..."